Sources/Controllers/MessagesViewController.swift

```diff
@@ -141,6 +141,20 @@ open class MessagesViewController: UIViewController {
             let trailing = messagesCollectionView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
             NSLayoutConstraint.activate([top, bottom, trailing, leading])
         }
+        adjustScrollViewInset()
+    }
+    
+    @objc
+    private func adjustScrollViewInset() {
+        if #available(iOS 11.0, *) {
+            // No need to add a to the top contentInset
```

Comment:
Nit: this sentence doesn't make sense. Change it to `no need to add to the top contentInset` maybe?